assets/js/recommender.js

```diff
@@ -0,0 +1,35 @@
+'use strict'
+
+let NUM_RECOMMENDED_PAGES = 5
+
+let pageUrl = document.getElementById('full-page-url').innerHTML
+
+let param = {
+  url: unescape(window.btoa(pageUrl))
```

Comment:
I'm not sure why we're unescaping the value from `window.btoa()` since the result should be guaranteed to be a base64 ASCII string (besides errors). That said [MDN](https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/btoa#Unicode_strings) does seem to suggest running `unescape()` and `encodeURIComponent()` to the value going into `window.btoa()` (i.e. `pageUrl`) since it can't handle Unicode values which are valid URL characters

Reply:
Makes sense! Edited such that the unescape happens before converting to base64 as per MDN recommendations